components/edit-collective/EditCollectiveArchive.js

```diff
@@ -83,7 +85,7 @@ const ArchiveCollective = ({ collective, archiveCollective, unarchiveCollective
             values={{ type: collectiveType.toLowerCase() }}
             id="collective.archive.description"
             defaultMessage={
-              'Archiving a {type} means it will visually appear inactive and no new activity will be allowed.'
+              'Archiving {type} means it will visually appear inactive and no new activity will be allowed.'
```

Comment:
`type` won't be translated. You can try this instead (need to remove `toLowerCase`):

```suggestion
              'Archiving {type, select, EVENT {this event}  COLLECTIVE {this collective} ORGANIZATION {this organization} other {this account}} means it will visually appear inactive and no new activity will be allowed.'
```